app/controllers/roles_controller.rb

```diff
@@ -0,0 +1,51 @@
+class RolesController < ApplicationController
```

Comment:
Should inherit from AuthenticatedController

Reply:
Done.